lib/active_triples/data_model.rb

```diff
@@ -0,0 +1,20 @@
+module ActiveTriples
+  ##
+  # Super class which provides a simple property DSL for defining property ->
+  # predicate mappings.
+  class DataModel
```

Comment:
I'm inclined to tame the property interface. :)

My concern with `DataModel` is that it suggests that this is the focal point for defining models.  That strikes me as confusing, but I'm open to argument.